modules/View.js

```diff
@@ -93,30 +126,31 @@
 
     /**
      * Hotswap rendering system reroute method.
+     * triggers templateRenderTiming on view to track render times
      * @method templateRender
      * See Torso.templateRenderer#render for params
      */
     templateRender: function(el, template, context, opts) {
       // Detach just this view's child views for a more effective hotswap.
       // The child views will be reattached by the render method.
+      var pageName = null;
```

Comment:
Move the comment about detaching child view to above where you detach child views.